src/components/utils/index.js

```diff
@@ -30,18 +30,29 @@ let isIOS = (/iPad|iPhone|iPod/.test(navigator.platform || '') || (navigator.pla
 export const isDesktop = !(/Android|webOS|BB10|BlackBerry|IEMobile|Opera Mini|Mobile|mobile/i.test(navigator.userAgent || '')) && !isIOS
 
 // To detect hybrid desktop/mobile devices which have a rear facing camera such as the Surface
-export async function isHybrid() {
+export async function isHybrid(facingMode) {
```

Comment:
Would be better to set a default parameter value since `facingMode` is used in the `getUserMedia` video constraint object.

Reply:
Fixed